CHANGELOG.D/128.feature

```diff
@@ -0,0 +1,2 @@
+Added --arg option to `neuro-flow bake` command that allows to pass arguments to `args` section
```

Comment:
Is the description up to date?

Reply:
It PR dependency issue. I'll fix this after rebase.

Reply:
Rebased.